src/ts/canvas.ts

```diff
@@ -4,8 +4,8 @@ export const canvasContext = canvas.getContext('2d')!;
 export const BACKGROUND_COLOR = '#121212';
 
 export function resizeCanvas() {
-  canvas.width = window.innerWidth;
-  canvas.height = window.innerHeight;
+  canvas.width = 1280;
+  canvas.height = 720;
```

Comment:
You don't need this function anymore. Can you just set it directly on the canvas element itself or if you need js, then just set it up once in bootstrap no need to run it on every resize.